test/quantization/core/test_docs.py

```diff
@@ -108,21 +109,23 @@ def _test_code(self, code, global_inputs=None):
             expr = compile(code, "test", "exec")
             exec(expr, global_inputs)
 
-    @unittest.skipIf(IS_ARM64, "Not working on arm")
+
     def test_quantization_doc_ptdq(self):
         path_from_pytorch = "docs/source/quantization.rst"
         unique_identifier = "PTDQ API Example::"
-        code = self._get_code(path_from_pytorch, unique_identifier)
-        self._test_code(code)
+        with override_quantized_engine("qnnpack") if IS_ARM64 else contextlib.nullcontext():
```

Comment:
If we do this for every test in here, should we do it in the `setUp` function instead?